projects/smoke_steering/src/main.c

```diff
@@ -9,17 +9,17 @@
 #include "wait.h"
 
 #define HORN_GPIO_ADDR \
+  { .port = GPIO_PORT_A, .pin = 4 }
+#define RADIO_PTT_GPIO_ADDR \
   { .port = GPIO_PORT_B, .pin = 1 }
-#define LANE_ASSIST_GPIO_ADDR \
```

Comment:
This should be okay. I assume we won't be adding lane assist. Its just strange since it seems to show up in the board
![image](https://user-images.githubusercontent.com/37023871/126916118-6937131c-2863-4ee2-a1d0-61531755f201.png)

Reply:
Lane assist is "radio push-to-talk" now I believe, it was already changed in steering but this is just to make it match in smoke_steering.